server/repository/src/db_diesel/stock_line_row.rs

```diff
@@ -40,6 +40,7 @@ joinable!(stock_line -> barcode (barcode_id));
 #[table_name = "stock_line"]
 pub struct StockLineRow {
     pub id: String,
+    #[column_name = "item_link_id"]
```

Comment:
This is a very subtle but critical change that makes this PR work - the struct and repository API remains `item_id`, but the table schema actually stores the value in column `item_link_id`.